lib/vm/src/instance/mod.rs

```diff
@@ -1141,6 +941,13 @@ impl InstanceHandle {
         self.instance().as_ref().vmctx_ptr()
     }
 
+    /// Return a reference to the `VMOffsets` to get offsets in the
+    /// `Self::vmctx_ptr` region. Be careful when doing pointer
+    /// arithmetic!
```

Comment:
I don't know if we need to document anything here or if this changes anything in regards to what is public / what we're saying we won't break, but something to keep in mind here!